public/js/cita.json

```diff
@@ -4101,21 +4021,21 @@
   },
   {
     "title": "Kirkby-in-Ashfield",
-    "address": "Portland Street\nKirkby-in-Ashfield\nNottinghamshire\nNG17 7AE",
+    "address": "Ashfield Health and Wellbeing Centre\nPortland Street\nKirby in Ashfield\nNottinghamshire\nNG17 7AE",
```

Comment:
The title says "Kirkby-in-Ashfield" but in the address we say "Kirby in Ashfield" – no dashes and a missing 'k'. Based on their website I think the value we had before was correct - http://ashfieldca.org.uk

Are we sure the new lat and long are required given the address is essentially the same, the only new detail is the name of the building.